VideoWeb/VideoWeb/ClientApp/src/app/services/user-media.service.ts

```diff
@@ -33,7 +33,7 @@ export class UserMediaService {
     get connectedMicrophoneDevices$(): Observable<UserMediaDevice[]> {
         return this.connectedDevicesSubject.pipe(
             map(devices => {
-                return devices.filter(x => x.kind === 'audioinput' && x.deviceId !== 'communications');
+                return devices.filter(x => x.kind === 'audioinput');
```

Comment:
Hi Rob, why deviceId !== 'communications' was removed, I guess this might cause duplicate audio device when call enumerateDevices()

Reply:
Hey Zijian, it is possible however, Abbey found that his laptop Mic was not showing up and it because it was filtered out using that.

Reply:
![image](https://user-images.githubusercontent.com/47365487/133981864-4f9c005c-b5d3-4fdd-9fee-84b36de39ea9.png)

Reply:
hey Rob I just pulled your branch and in the dropdown list I have duplicated Poly BT600

Reply:
Yeah this was expected but I am guessing that it was filtering a device with the label "communications"

Reply:
Going to test a different approach where if no over devices are available after filtering it will return the comms and default devices.